cabot/cabotapp/migrations/0005_auto_20170818_1202.py

```diff
@@ -15,17 +16,25 @@ def move_old_jenkins_checks(apps, schema_editor):
     JenkinsCheck = apps.get_model("cabotapp", "JenkinsCheck")
     JenkinsConfig = apps.get_model("cabotapp", "JenkinsConfig")
 
+    if not JenkinsStatusCheck.objects.exists():
+        return
```

Comment:
So does this condition work? Given how `JenkinsStatusCheck.objects.all()` returns everything ?

Reply:
good point